src/aleph/vm/orchestrator/resources.py

```diff
@@ -122,6 +123,18 @@ async def about_system_usage(_: web.Request):
     return web.json_response(text=usage.json(exclude_none=True))
 
 
+@cors_allow_all
+async def about_certificates(request: web.Request):
+    """Public endpoint to expose platform certificates for confidential computing."""
+
+    if not settings.ENABLE_CONFIDENTIAL_COMPUTING:
+        return web.HTTPBadRequest(reason="Confidential computing setting not enabled on that server")
+
+    sev_client: SevClient = request.app["sev_client"]
+
+    return web.FileResponse(await sev_client.get_certificates())
```

Comment:
Is any error expected to be potentially raised by  `await sev_client.get_certificates()` ? That would result in an error 500 (and a log in Sentry).

Reply:
Shouldn't be, because it only executes `sevctl` command, that is already checked on the supervisor initialization, and also the folder is already created at the start.